demo/src/test/java/com/example/test/pages/KBBMOOCProject/SearchCarPage.java

```diff
@@ -0,0 +1,33 @@
+package com.example.test.pages.KBBMOOCProject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+
+
+public class SearchCarPage  extends KbbBasePage{
+
+    public SearchCarPage(WebDriver driver) {
+        super(driver);
+    }
+   
+ 
+WebElement selectYear = keyword.findWebElementByXpath("//select[@placeholder='Year']");
+    WebElement selectMake = keyword.findWebElementByXpath("//div[@class='make']//select[@placeholder='Make']");
+    WebElement selectModel = keyword.findWebElementByXpath("//div[@class='model']//select[@placeholder='Model']");
+    WebElement inputZip = keyword.findWebElementByXpath("//input[@type='tel']");
+    WebElement btnGetReview = keyword.findWebElementByXpath("//form//button[@type='submit']");
```

Comment:
should write the web element locator in separate line then call into a method/function